components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -73,33 +75,9 @@ export default {
 			},
 			control: "boolean",
 		},
-		isHovered: {
-			name: "Hovered",
-			type: { name: "boolean" },
-			table: {
-				type: { summary: "boolean" },
-				category: "State",
-			},
-			control: "boolean",
-		},
-		isFocused: {
-			name: "Focused",
-			type: { name: "boolean" },
-			table: {
-				type: { summary: "boolean" },
-				category: "State",
-			},
-			control: "boolean",
-		},
-		isActive: {
-			name: "Active",
-			type: { name: "boolean" },
-			table: {
-				type: { summary: "boolean" },
-				category: "State",
-			},
-			control: "boolean",
-		},
+		isHovered: { table: { disable: true } },
+		isFocused: { table: { disable: true } },
+		isActive: { table: { disable: true } },
```

Comment:
@jawinn curious if you think this could be a solve to [your callout](https://github.com/adobe/spectrum-css/pull/2651#pullrequestreview-2001267993) in my switch PR. This way, we hide these controls from the users so the classes don't get added in the markup unless the testing preview is turned on. So we maintain the current Chromatic coverage and users can still manually try out those hover and focus and active states.

I still don't think it's ideal to be adding classes that we don't support in the CSS and I'd be hesitant to start adding this on other PRs, but it solves the class problem here without changing coverage. My hope is that we can get some native focus and hover coverage in Chromatic without the classes. I think we have one or two cards on the board/in the backlog for that

Reply:
I think this is a good solution!

This is probably the best option for now, AFAIK. The currently documented Chromatic way to to do [focus and hover](https://www.chromatic.com/docs/hoverfocus/) uses `play` functions, which won't work well in the kitchen sink views where we can have multiple things that should show hover and focus at the same time.